client/src/partials/templates/grid/voucherType.tmpl.html

```diff
@@ -0,0 +1,7 @@
+<div style="padding: 5px;">
```

Comment:
I think @sfount created a class `ui-grid-cell-contents` to provide padding and proper overflow for grid cells.  See https://github.com/IMA-WorldHealth/bhima-2.X/pull/537#discussion_r68903437 for more information.  You should use that on all these grid cells.